sched/task/task_activate.c

```diff
@@ -94,4 +94,11 @@ void nxtask_activate(FAR struct tcb_s *tcb)
     }
 
   leave_critical_section(flags);
+
+#if CONFIG_TASK_NAME_SIZE>0
+  sinfo("%s pid=%d,TCB=%p\n", tcb->name,
```

Comment:
Is the printed log information enough to know whether it is active or exit?

Reply:
These information should be enough for high level judgements, there are at most two logs for one task: one for first run and one for end of life. The current information are more than most logs in "up_exit" where only TCB address is printed. We sure can enrich later but it is fine for now.